code/game/objects/items/weapons/material/kitchen.dm

```diff
@@ -192,3 +192,67 @@
 		user.Paralyse(2)
 		return
 	return ..()
+
+/obj/item/material/kitchen/utensil/knife/pocketbutterknife
```

Comment:
This should be a generic item type first, and then a heph subtype.

generic `utensil/knife/pocketbutterknife` and then `utensil/knife/pocketbutterknife/heph` for the heph variant

Reasoning:
- It is not immediately obvious that a pocket butter knife is heph-made and for heph employees. People are going to map it in or spawn it, without knowing it is heph, where it doesn't make sense to have a heph knife.
- If someone wanted to add another pocket butter knife variant (generic/zavod/whatever), it would be weird for it to be a subtype of a heph knife. Or they would have to change the type of the heph knife to be some subtype, but that would be messy.
- It's just, more correct and better for future maintenance to do it like this.